pkgs/development/haskell-modules/lib.nix

```
@@ -232,6 +232,31 @@ rec {
     fixupPhase = ":";
   });
 
+  /* Create a documentation tarball suitable for uploading to Hackage instead
+     of building the package.
+   */
+  documentationTarball = pkg:
+    pkgs.lib.overrideDerivation pkg (drv: {
+      name = "${drv.pname}-${drv.version}-docs";
```

This is a pretty small nitpick, but couldn't this be

```suggestion
      name = "${drv.name}-docs";
```